code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1231,3 +1231,20 @@ datum/reagent/romerol
 	// Silently add the zombie infection organ to be activated upon death
 	new /obj/item/organ/body_egg/zombie_infection(H)
 	..()
+
+/datum/reagent/growthserum
+	name = "growth serum"
+	id = "growthserum"
+	description = "A commercial chemical designed to help older men in the bedroom."//not really it just makes you a giant
+	color = "#ff0000"//strong red. rgb 255, 0, 0
+
+/datum/reagent/growthserum/reaction_mob(mob/H, method=INGEST, reac_volume)
```

Comment:
don't typecast this for all mobs, and certainly not as mob/H (standard is M).

I'd recommend just making this something that happens `on_mob_life`, but only actually happens on the first cycle, then the reagent deleted thing will work.